content/en/docs/concepts/scheduling/scheduling-framework.md

```diff
@@ -228,8 +230,8 @@ type PreFilterPlugin interface {
 # Plugin Configuration
 
 Plugins can be enabled in the scheduler configuration. Also, default plugins can
-be disabled in the configuration. In 1.15, there are no default plugins for the
-scheduling framework.
+be disabled in the configuration. Starting from 1.18, there are a number of [default 
+plugins](/docs/reference/scheduling/profiles/#scheduling-plugins) enabled for the scheduling framework.
 
 The scheduler configuration can include configuration for plugins as well. Such
```

Comment:
All of this should go to the new doc.
Could you rebase on top of my PR?

Reply:
But other than that, this section should have some notes about adding plugins to the out-of-tree registry.

Reply:
> All of this should go to the new doc.
> Could you rebase on top of my PR?

Nice, let me simply link to the profile doc. Do you want me to remove all the samples below, and then you can include them in `docs/reference/scheduling/profiles.md`?

> But other than that, this section should have some notes about adding plugins to the out-of-tree registry.

Reworded to mention that "In addition to default plugins, you can also implement your own scheduling plugins and get them configured along with default plugins."

Reply:
I'll do a followup PR once the previous one gets merged.